database/src/pool/postgres.rs

```diff
@@ -387,6 +387,29 @@ static MIGRATIONS: &[&str] = &[
     r#"
     ALTER TABLE collector_config ADD COLUMN commit_sha TEXT NULL;
     "#,
+    r#"
+    CREATE TABLE IF NOT EXISTS error_new (
+        id      SERIAL PRIMARY KEY,
+        aid     INTEGER NOT NULL,
```

Comment:
```suggestion
    aid     INTEGER  NOT NULL REFERENCES artifact(id) ON DELETE CASCADE ON,
```
We want to keep the FK constraint, and also remove all errors when an artifact is purged from the DB.